libs/scraper-archive/src/snapshots.ts

```
@@ -0,0 +1,90 @@
+  const snapshotFile = path.join(snapshotFolder, `${element}-elm-${timestamp.getTime()}.json`);
+
+  const data = getWriteData(original, found, candidates);
+  writeFileSync(snapshotFile, JSON.stringify(data, null, 2));
+}
+
+  // Useful things in a snapshot:
+  //  - found element
+  //  - expected results (if not found)
+  //  - top 10 candidates
+function getWriteData(original: TestElmData, found: FoundElement, candidates: FoundElement[]): SnapshotData {
+  const matched = candidates.find(c => (
```

ESLint

[error] Expected parentheses around arrow function argument.

(arrow-parens)